plugin/bitrise/environ.go

```diff
@@ -31,6 +31,21 @@ func Environ(src []string) []string {
 		}
 	}
 
+	bitriseDefaults := map[string]string{
+		"BITRISEIO_FINISHED_STAGES":      "",
+		"BITRISE_DEPLOY_DIR":             "",
+		"BITRISE_APP_TITLE":              "",
+		"BITRISE_APP_URL":                "",
+		"BITRISE_APP_SLUG":               "",
+		"BITRISE_PROVISION_URL":          "",
+		"BITRISE_CERTIFICATE_URL":        "",
+		"BITRISE_CERTIFICATE_PASSPHRASE": "",
+	}
+
+	for key, defaultValue := range bitriseDefaults {
+		dst[key] = firstMatch(dst, key, defaultValue)
```

Comment:
firstMatch takes a map and list of keys. Since here `defaultValue` is always empty there is no need to send it.
Just keep it as
```
for key, _ := range bitriseDefaults {
		dst[key] = firstMatch(dst, key)
}
```